alerts/ui/src/components/CustomAppShell.jsx

```diff
@@ -62,10 +47,10 @@ const CustomAppShell = ({ children }) => {
 
   return (
     <AppShell
-      pageHeader={pageHeader}
+      pageHeader={<PageHeader heading="Supernova" />}
       embedded={embedded}
-      topNavigation={topNavigation}
     >
+      {topNavigation}
```

Comment:
The way the `topNavigation` was passed as a prop to the Appshell was correct. If this didn't work for you then we need to upgrade `juno-ui-components` to the latest version